.circleci/config.yml

```diff
@@ -605,6 +653,51 @@ jobs:
         path: /tmp/test-reports
     - store_artifacts:
         path: /tmp/test-reports
+  build-darwin-binaries:
+    macos:
+      xcode: 11.3.1
+    working_directory: ~/go/src/github.com/hashicorp/nomad
+    steps:
+    - checkout
+    - run:
+        command: echo 'export PATH="${GOPATH}/bin:${HOME}/goinstall/go/bin:$PATH"' >> ${BASH_ENV}
+        name: configure PATH
+    - run:
+        command: |
+          set -x
+
+          os="linux"
+          if [[ "$OSTYPE" == "darwin"* ]]
+          then
+            os="darwin"
+          fi
+
+          echo installing golang ${GOLANG_VERSION}
+          sudo rm -rf ~/goinstall/go
+          sudo mkdir -p ~/goinstall
+          curl -SL --fail -o /tmp/golang.tar.gz https://dl.google.com/go/go${GOLANG_VERSION}.${os}-amd64.tar.gz
+          sudo tar -C ~/goinstall -xzf /tmp/golang.tar.gz
+          rm -rf /tmp/golang.tar.gz
+        name: install golang
+    - run:
+        command: source ${BASH_ENV} && make deps
+    - run:
+        command: brew install protobuf
+    - run:
+        command: sudo -E PATH="$GOPATH/bin:${HOME}/goinstall/go/bin:$PATH" make generate-structs
+    - run:
+        command: source ${BASH_ENV} && make pkg/darwin_amd64.zip
+    - store_artifacts:
+        destination: /builds/nomad_darwin_amd64.zip
+        path: pkg/windows_amd64.zip
```

Comment:
is this the correct path? if its arbitrary maybe still rename to darwin_amd64

Reply:
🤦‍♂ Oops